packages/default/scss/dialog/_theme.scss

```diff
@@ -3,6 +3,7 @@
     $dialog-titlebar-border-color: inherit !default;
     $dialog-titlebar-bg: $accent !default;
     $dialog-titlebar-text: $accent-contrast !default;
+    $hover-active-selector: ":hover, .k-state-hover, :active, .k-state-active";
```

Comment:
I don't think that's a pattern that we should use. It hides away the selector complexity.

Also, [according to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:not#Browser_compatibility), the `:not(.foo, .bar)` is supported only in Safari 9+ at the moment, so probably we don't need these rules?